src/zenml/materializers/special_string_materializer.py

```diff
@@ -20,24 +20,25 @@
 from zenml.io import fileio
 from zenml.logger import get_logger
 from zenml.materializers.base_materializer import BaseMaterializer
-from zenml.types import HTMLString, MarkdownString
+from zenml.types import CSVString, HTMLString, MarkdownString
 
 logger = get_logger(__name__)
 
 
-HTML_OR_MARKDOWN = Union[HTMLString, MarkdownString]
+SPECIAL_STRINGS = Union[CSVString, HTMLString, MarkdownString]
```

Comment:
How about 'STRUCTURED_STRINGS' instead? Somehow SPECIAL could be lots of things, like those containing special characters etc.